develop/api_guc.rst

```diff
@@ -99,6 +99,35 @@ Citus honors only a whitelisted subset of the options, namely:
 
 *(† = subject to the runtime presence of optional PostgreSQL features)*
 
+.. _override_table_visibility:
```

Comment:
Currently, the MX docs say `If you connect to the data URL using psql and run \d, then you will see all the distributed tables and some of the shards. ` We should mention this guc there as well.

Reply:
Good point, thanks.